hawc/apps/assessment/templates/assessment/assessmentvalue_detail.html

```diff
@@ -65,6 +65,14 @@ <h3>Assessment Value for {{object.assessment}}</h3>
       {% endfor %}
     </tbody>
   </table>
+  {% if obj_perms.edit and value_warning %}
+    {% alert warning %}
+      The POD/UF = Value is different than what is reported.
+      ({{object.pod_value|floatformat:'-3'}}/{{object.uncertainty|floatformat:'-3'}} = {{value_warning|floatformat:'-3'}}, which is different than
```

Comment:
I didn't know about this one until now, but if you use [stringformat](https://docs.djangoproject.com/en/5.0/ref/templates/builtins/#stringformat) instead of `floatformat`, you can use a lot more options; honestly the floatformat seems pretty limited for our scientific use case https://docs.python.org/3/library/string.html#format-string-syntax